src/client/dfs/dfs_internal.h

```diff
@@ -66,7 +66,11 @@ dfs_get_sb_layout(daos_key_t *dkey, daos_iod_t *iods[], int *akey_count,
 void
 dfs_free_sb_layout(daos_iod_t *iods[]);
 
-/** as dfs_open() but takes a stbuf to be populated if O_CREATE is specified */
+/** as dfs_open() but takes a stbuf to be populated if O_CREATE is specified
+ *
+ * If O_CREATE is set then entry->uid and entry->gid are popouated with the desired user,
```

Comment:
i am fine with change as this is an internal API.

Reply:
Yes, it's not a great interface but this is why it's internal.